docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md

```diff
@@ -90,22 +67,37 @@ To publish an add-in to an existing app catalog, complete the following steps.
 
     Content and task pane add-ins in this catalog are now available from the  **Office Add-ins** dialog box. To access them, choose **My Add-ins** on the **Insert** tab, and then choose **MY ORGANIZATION**.
 
-## End user experience with the add-in catalog
+## Publish an add-in to an app catalog with on-premises SharePoint Server
+
+1. Open the **Central Administration** page.
+2. In the left task pane, choose **Apps**.
+3. On the **Apps** page, under **App Management**, choose **Manage App Catalog**.
+4. On the **Manage App Catalog** page, make sure you have the right web application selected in the **Web Application** Selector.
+5. Choose the URL under the **Site URL** to open the app catalog site.
+6. Choose **Distribute apps for Office**.
+7. In the **Apps for Office** page, choose **New**.
+8. In the **Add a document** dialog, select the **Choose Files** button.
+9. Locate and specify the [manifest](../develop/add-in-manifests.md) file to upload and choose **Open**.
+10. In the **Add a document** dialog, choose **OK**.
+
+    Content and task pane add-ins in this catalog are now available from the  **Office Add-ins** dialog box. To access them, choose **My Add-ins** on the **Insert** tab, and then choose **MY ORGANIZATION**.

```

Comment:
Is the indentation at the start of this line intentional, or should it be removed (so that the text is left-aligned)?

Also, suggest that you clarify this text to indicate where the **Office Add-ins** dialog box exists, and where you access them. I assume you're talking about the Office application (e.g., Word, Excel, etc.), but that's not clear to me, as currently written.

Reply:
Okay, did some large reorganization to make it clear when steps are for online, vs desktop, and how to insert an add-in once you are connected to the catalog. Hope this works.